Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,8 @@
+function calculateFrequency(changes) {
```

Comment:
I think the requirement is to write a JS script that can read the numbers from `input.txt` and sum them up.
Can you write the script so that it can calculate the sum without first storing all the numbers in an array first?

Just to implement a sum() function would not be much of a challenge. Would it?